doc/source/suse-ecp/ose-localhost.rst

```diff
@@ -85,6 +85,18 @@ venv in |socok8s_workspace_default|\ `/.ansiblevenv`.
       :code:
 
 
+For CasSP 4.x deployment, skuba (https://github.com/SUSE/skuba) and terraform
+packages are required. Currently deployment steps automatically configure
+related package repositories if ansible runner host (the host where ansible is
```

Comment:
nit: configure related packages repositories and installs the appropriate packages

(it's easier to read that way)